src/main/java/slimeknights/tconstruct/common/config/Config.java

```diff
@@ -46,6 +47,10 @@ private Config() {
   public static String[] craftingStationBlacklist = new String[] {
       "de.ellpeck.actuallyadditions.mod.tile.TileEntityItemViewer"
   };
+  private static String[] orePreference = {
+      "minecraft",
+      "tconstruct"
```

Comment:
Should probably put some more defaults here so people don't get chisel for copper etc. by default. TE always is a good fit ("thermalfoundation" I think), and immersive engineering.

Reply:
Industrial Craft would also be a good fit in this list